plugins/weather/src/test/java/de/unistuttgart/iaas/amyassist/amy/plugin/weather/WeatherRestTest.java

```diff
@@ -76,13 +79,13 @@ public void setUp() {
 	private void createDay() {
 		FIODataPoint p = Mockito.mock(FIODataPoint.class);
 		Mockito.when(p.summary()).thenReturn("Clear throughout the day");
-		Mockito.when(p.precipProbability()).thenReturn((double) 0);
+		Mockito.when(p.precipProbability()).thenReturn(Double.valueOf(0));
```

Comment:
use `0.0`

Reply:
I know but then we would get warnings like "The expression of type double is boxed into Double". I think it was allowed to have this sort of warnings but if I do it with valueOf the warnings fall away. Should I change it anyway?

Reply:
change the eclipse settings to ignore the warning

Reply:
ok